docs/reference/query-dsl/filters/terms-filter.asciidoc

```diff
@@ -19,71 +19,6 @@ The `terms` filter is also aliased with `in` as the filter name for
 simpler usage.
 
 [float]
-==== Execution Mode
-
-The way terms filter executes is by iterating over the terms provided
-and finding matches docs (loading into a bitset) and caching it.
-Sometimes, we want a different execution model that can still be
-achieved by building more complex queries in the DSL, but we can support
-them in the more compact model that terms filter provides.
-
-The `execution` option now has the following options :
-
-[horizontal]
-`plain`:: 
-    The default. Works as today. Iterates over all the terms,
-    building a bit set matching it, and filtering. The total filter is
-    cached.
-
-`fielddata`::
-    Generates a terms filters that uses the fielddata cache to
-    compare terms.  This execution mode is great to use when filtering
-    on a field that is already loaded into the fielddata cache from 
-    aggregating, sorting, or index warmers.  When filtering on
-    a large number of terms, this execution can be considerably faster
-    than the other modes.  The total filter is not cached unless
-    explicitly configured to do so.
-
-`bool`:: 
-    Generates a term filter (which is cached) for each term, and
-    wraps those in a bool filter. The bool filter itself is not cached as it
-    can operate very quickly on the cached term filters.
-
-`and`:: 
-    Generates a term filter (which is cached) for each term, and
```

Comment:
That said, it is a corner case... I can't imagine that many people are using it.